app/index.html

```diff
@@ -1,105 +1,108 @@
 <!doctype html>
 <html lang="">
-  <head>
-    <meta charset="utf-8">
-    <meta name="description" content="">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>ec2pricing</title>
 
-    <link rel="apple-touch-icon" href="apple-touch-icon.png">
-    <!-- Place favicon.ico in the root directory -->
+<head>
+  <meta charset="utf-8">
+  <meta name="description" content="">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>ec2pricing</title>
 
-    <!-- build:css styles/vendor.css -->
-    <!-- bower:css -->
-    <link rel="stylesheet" href="/bower_components/bootstrap-select/dist/css/bootstrap-select.css" />
-    <!-- endbower -->
-    <!-- endbuild -->
+  <link rel="apple-touch-icon" href="apple-touch-icon.png">
+  <!-- Place favicon.ico in the root directory -->
 
-    <!-- build:css styles/main.css -->
-    <link rel="stylesheet" href="styles/main.css">
-    <!-- endbuild -->
+  <!-- build:css styles/vendor.css -->
+  <!-- bower:css -->
+  <link rel="stylesheet" href="/bower_components/bootstrap-select/dist/css/bootstrap-select.css" />
+  <!-- endbower -->
+  <!-- endbuild -->
 
-  </head>
-  <body>
-    <!--[if lt IE 10]>
+  <!-- build:css styles/main.css -->
+  <link rel="stylesheet" href="styles/main.css">
+  <!-- endbuild -->
+
+</head>
+
+<body>
+  <!--[if lt IE 10]>
       <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
     <![endif]-->
 
-    <div class="container">
-      <div class="header">
-        <h3 class="text-muted">EC2 Spot Pricing</h3>
-      </div>
+  <div class="container">
+    <div class="header">
+      <h3 class="text-muted">EC2 Spot Pricing</h3>
+    </div>
 
-      <div class="pricing-table">
-        <div class="row">
-          <div class="col-xs-12 col-md-6 col-lg-4">
-            <div class="form-group">
-              <label for="region-select">Region</label>
-              <select class="form-control selectpicker" id="region-select" data-live-search="true">
+    <div class="pricing-table">
+      <div class="row">
+        <div class="col-xs-12 col-md-6 col-lg-4">
+          <div class="form-group">
+            <label for="region-select">Region</label>
+            <select class="form-control selectpicker" id="region-select" data-live-search="true">
```

Comment:
Please don't add style changes with your own change. It becomes difficult to review your change